### ISSUE TITLE
Fine-tune model

### ISSUE THREAD

**Issue by galfaroth** [haoheliu/AudioLDM#19]
Would it be able to fine-tune the model with folder of own sounds like e.g. kick samples?

**Comment by haoheliu** [comment]
@galfaroth Yeah definitely. The fine-tuning code will be available after paper acceptance (if that happens).

**Comment by galfaroth** [comment]
Crossing fingers. I'm so excited by this work!

**Comment by bitnom** [comment]
I must fine-tune this. must must must

**Comment by jmren168** [comment]
> @galfaroth Yeah definitely. The fine-tuning code will be available after paper acceptance (if that happens).

@haoheliu 
Hi, do you still paly to make the fine-tuning code available? Mank thanks.

**Comment by Tortoise17** [comment]
Is there code now available for fine tune? Would be a kind help.

**Comment by Tortoise17** [comment]
@haoheliu  Hello. As the work has been published, now will you release the pipeline to finetune the model for own data or so?
